utils/browser_history.tsx

```diff
@@ -27,7 +37,7 @@ window.addEventListener('message', ({origin, data: {type, message = {}} = {}} =
 
 export const browserHistory = {
     ...b,
-    push: (path, ...args) => {
+    push: (path: any, ...args: string[]) => {
         if (isDesktop) {
```

Comment:
i could't give **path** a type coz it can either be an **object** or **string**, but how can that be

Reply:
```suggestion
        push: (path: string | Record<string, string>, ...args: string[]) => {
```
Maybe this will help. I think you will need to figure out the object type.